rust/protocol/src/sealed_sender.rs

```diff
@@ -853,6 +978,21 @@ mod sealed_sender_v2 {
         Ok(result)
     }
 
+    /// Compute an [authentication tag] for the encrypted result `encrypted_message_key` produced by
+    /// [apply_agreement_xor] using a secret derived from the sender's private identity key
```

Comment:
Rust convention seems to be to use code voice even within links (and rustdoc accepts it with no additional annotation): ``[`apply_agreement_xor`]``. There are a *bunch* of places to change this, sorry!

Reply:
Not a problem, emacs makes it easy!